packages/bpk-component-accordion/src/BpkAccordionItem.scss

```diff
@@ -74,6 +74,10 @@
     display: inline-block;
   }
 
+  &__leading-icon {
+    margin-right: $bpk-spacing-xs;
```

Comment:
This may need to switch to `margin-left` when RTL is enabled. We have a mixin that can do it automatically: https://backpack.github.io/sassdoc/#mixin-bpk-margin-trailing

Reply:
Ah yes! Good spot!